try:
    pass
except:
    pass
finally:
    match name_3:
        case None:
            pass
{name_5}

# output:
# python: Python/flowgraph.c:511: no_redundant_jumps: Assertion `0' failed.
